apps/server/src/shared/domain/entity/dashboard.entity.ts

```diff
@@ -217,8 +217,21 @@ export class DashboardEntity {
 		return element;
 	}
 
-	moveElement(from: GridPositionWithGroupIndex, to: GridPositionWithGroupIndex): GridElementWithPosition {
+	moveElement(
+		from: GridPositionWithGroupIndex,
+		to: GridPositionWithGroupIndex,
+		substituteCourses: ILearnroom[]
+	): GridElementWithPosition {
 		const elementToMove = this.getReferencesFromPosition(from);
+		this.checkIfMovedToSubstitute(elementToMove, substituteCourses);
```

Comment:
missing condition if substitueCourses is empty, to avoid executing code

Reply:
done